polaris-react/.storybook/GridOverlay/GridOverlay.css

```diff
@@ -15,8 +15,8 @@
   opacity: 0.25;
   pointer-events: none;
 }
-
-@media screen and (min-width: 768px) {
+/* #{$p-breakpoints-sm-up} */
+@media (min-width: 768px) {
```

Comment:
For these ones should we just use the new variable?

Reply:
We could but would have to convert this file to SCSS. @kyledurand I forget, was there a reason we wrote this in vanilla CSS?

Reply:
I guess storybook doesn't build scss without plugins... That makes sense.